sources/packages/backend/workflow/test/2022-2023/parttime-assessment/appeals/parttime-assessment-student-financial-information.e2e-spec.ts

```diff
@@ -97,6 +97,53 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-student-financia
     },
   );
 
+  it(
+    "Should have calculated student total income from the current year income and not from the the application data or the tax return income" +
```

Comment:
It needs a white space at the end of the first sentence, otherwise, it will be "incomewhen".